packages/crypto/src/MultisigSignature.ts

```diff
@@ -0,0 +1,121 @@
+import Address, { KeyTypes, MultisigAddress } from '@helium/address'
```

Comment:
These changes will need to be brought over to the `crypto-react-native` package if we want to support this in the helium app. I am ok with doing that as a separate PR if wanted though.